modules/consul-security-group-rules/README.md

```diff
@@ -3,7 +3,7 @@
 This folder contains a [Terraform](https://www.terraform.io/) module that defines the security group rules used by a 
 [Consul](https://www.consul.io/) cluster to control the traffic that is allowed to go in and out of the cluster. 
 
-Normally, you'd get these rules by default if you're using the [consul-cluster module](https://github.com/hashicorp/terraform-aws-consul/tree/master/MAIN.md), but if 
+Normally, you'd get these rules by default if you're using the [consul-cluster](https://github.com/hashicorp/terraform-aws-consul/tree/master/modules/consul-cluster), but if 
```

Comment:
This is actually a reference to `modules/consul-cluster`, so this change is correct.